plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/sql/SQLDialect.java

```diff
@@ -270,6 +270,10 @@ default boolean supportsAliasInConditions() {
         return true;
     }
 
+    default boolean supportsAliasForHavingSyntax() {
```

Comment:
Let's rename it to supportsAliasInHaving

Reply:
Done